client/landing/gutenboarding/components/domain-categories/style.scss

```diff
@@ -10,14 +10,14 @@
 		&:focus {
 			color: var( --studio-gray-100 );
 			box-shadow: none;
-			font-weight: 700;
+			font-weight: 600;
```

Comment:
FYI @dubielzyk, for us this will be changing selected category in domain modal to regular bold weight.